argilla/CHANGELOG.md

```diff
@@ -14,7 +14,27 @@ These are the section headers that we use:
 * "Security" in case of vulnerabilities.
 -->
 
-## [Unreleased]()
+## [2.0.1](https://github.com/argilla-io/argilla/compare/v2.0.0...releases/2.0.1)
```

Comment:
should we add contributors to the changelog? I believe we added @ contributor name before.

Reply:
Sounds good. Just community ones, or team as well?

Reply:
I would just go for community contributions.